definitions/json/CaseEventToFields.json

```diff
@@ -2745,11 +2745,21 @@
     "PageFieldDisplayOrder": 6,
     "ShowSummaryChangeOption": "Y"
   },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "CaseEventID": "et1Vetting",
+    "CaseFieldID": "trackType",
+    "FieldShowCondition": "et1JurisdictionCodeGeneralNotes=\"dummy dummy dummy dum dummy\"",
+    "DisplayContext": "READONLY",
+    "PageID": 5,
+    "PageDisplayOrder": 5,
+    "PageFieldDisplayOrder": 7
+  },
   {
     "CaseTypeID": "ET_EnglandWales",
     "CaseEventID": "et1Vetting",
     "CaseFieldID": "trackAllocation",
-    "FieldShowCondition": "et1SubstantiveDefectsGeneralNotes=\"dummy\"",
+    "FieldShowCondition": "et1SubstantiveDefectsGeneralNotes=\"dummy dummy dummy dum dummy\"",
```

Comment:
What I meant with my previous comment on this was not to set a text field as the show condition. The user can still type in dummy dummy dummy dum dummy and this field would appear. Maybe set the field for the condition as a YesNo field.